src/scripts/main.js

```diff
@@ -1,5 +1,34 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
+wall.addEventListener('click', e => {
+  const spiderHalf = spider.offsetWidth / 2;
+  const spiderSize = spiderHalf * 2;
+  const coordsX = e.offsetX;
+  const coordsY = e.offsetY;
+  const wallWidth = wall.clientWidth;
+  const wallHeight = wall.clientHeight;
+  let spiderCoordsX = coordsX - spiderHalf + 'px';
+  let spiderCoordsY = coordsY - spiderHalf + 'px';
```

Comment:
Відступи між змінними зайвими не будуть.
```suggestion
  const spiderHalf = spider.offsetWidth / 2;
  const spiderSize = spiderHalf * 2;
  
  const coordsX = e.offsetX;
  const coordsY = e.offsetY;
  
  const wallWidth = wall.clientWidth;
  const wallHeight = wall.clientHeight;
  
  let spiderCoordsX = coordsX - spiderHalf + 'px';
  let spiderCoordsY = coordsY - spiderHalf + 'px';
```